lib/parse_script_to_json.py

```diff
@@ -15,6 +15,7 @@ def parse_script_to_json(script_file, location=SCRIPT_DIR):
     replace = []
     keys_to_ignore = ["template"]
     urls = {}
+    values["retriever"] = "True"
```

Comment:
This change sets the retriever json files with `"retriever"= "True"` to help identify the right json to compile.

however, I did not see the need to also include it in the written `.py` file since we already have `#retirver ` tag in the `.py` files